src/types.ts

```diff
@@ -24,6 +24,6 @@ export interface Sender {
 }
 
 export interface PersistentStorage {
-  shift(cb: (err: Error | null, value?: unknown) => void): void;
-  push(value: unknown, cb: (err: Error | null, result?: boolean) => void): void;
+  shift(cb: (err: Error | null, value?: unknown[]) => void): void;
```

Comment:
What is this unknown type and why you need an array of it?, first time I see something like this that why I'm asking

Reply:
The persister doesn't care what type of object it is storing since I am not interacting with the object at all, just dumping / reading it from disk. Ideally it is just `unknown`, but it creates type conflicts w/ HttpSender expecting an array.